src/lucky/params.cr

```diff
@@ -492,4 +523,20 @@ class Lucky::Params
   private def query_params
     request.query_params
   end
+
+  private def get_all_json(key : String)
+    val = parsed_json[key]?
+    return if val.nil?
+
+    val.as_a?.try(&.map(&.to_s)) || [val.to_s]
+  end
+
+  private def get_all_params(params, key : String)
+    vals = params.fetch_all(key + "[]")
```

Comment:
You might ask, "What about if the user doesn't want to use the `key[]` pattern?"

I'm open to feedback but I think we're pretty clear in Lucky that we prefer convention with a backdoor and the backdoor in this scenario is to access the params directly: `params.from_query.fetch_all("keys")`